<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstrumentEditorADPCMForm</class>
 <widget class="QDialog" name="InstrumentEditorADPCMForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Sample</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="sampleGroupBox">
         <property name="title">
          <string>Sample</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="ADPCMSampleEditor" name="sampleEditor" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Envelope</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QGroupBox" name="envEditGroupBox">
         <property name="title">
          <string>Envelope</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QSpinBox" name="envNumSpinBox">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="prefix">
             <string notr="true">#</string>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="VisualizedInstrumentMacroEditor" name="envEditor" native="true"/>
          </item>
          <item row="0" column="1" alignment="Qt::AlignRight">
           <widget class="QLabel" name="envUsersLabel">
            <property name="text">
             <string>Users:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="envUsersLineEdit">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Arpeggio</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="arpEditGroupBox">
         <property name="title">
          <string>Arpeggio</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="5">
           <widget class="QComboBox" name="arpTypeComboBox"/>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="arpTypeLabel">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QSpinBox" name="arpNumSpinBox">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="prefix">
             <string notr="true">#</string>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="6">
           <widget class="ArpeggioMacroEditor" name="arpEditor" native="true"/>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" alignment="Qt::AlignRight">
           <widget class="QLabel" name="arpUsersLabel">
            <property name="text">
             <string>Users:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="arpUsersLineEdit">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Pitch</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="ptEditGroupBox">
         <property name="title">
          <string>Pitch</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="1" alignment="Qt::AlignRight">
           <widget class="QLabel" name="ptUsersLabel">
            <property name="text">
             <string>Users:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="ptTypeLabel">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="ptTypeComboBox"/>
          </item>
          <item row="1" column="0" colspan="6">
           <widget class="VisualizedInstrumentMacroEditor" name="ptEditor" native="true"/>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QSpinBox" name="ptNumSpinBox">
            <property name="prefix">
             <string notr="true">#</string>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="ptUsersLineEdit">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Panning</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="panEditGroupBox">
         <property name="title">
          <string>Panning</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QSpinBox" name="panNumSpinBox">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="prefix">
             <string notr="true">#</string>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="PanMacroEditor" name="panEditor" native="true"/>
          </item>
          <item row="0" column="1" alignment="Qt::AlignRight">
           <widget class="QLabel" name="panUsersLabel">
            <property name="text">
             <string>Users:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="panUsersLineEdit">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VisualizedInstrumentMacroEditor</class>
   <extends>QWidget</extends>
   <header>gui/instrument_editor/visualized_instrument_macro_editor.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArpeggioMacroEditor</class>
   <extends>QWidget</extends>
   <header>gui/instrument_editor/arpeggio_macro_editor.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanMacroEditor</class>
   <extends>QWidget</extends>
   <header>gui/instrument_editor/pan_macro_editor.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ADPCMSampleEditor</class>
   <extends>QWidget</extends>
   <header>gui/instrument_editor/adpcm_sample_editor.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>envEditGroupBox</tabstop>
  <tabstop>envNumSpinBox</tabstop>
  <tabstop>arpEditGroupBox</tabstop>
  <tabstop>arpNumSpinBox</tabstop>
  <tabstop>arpTypeComboBox</tabstop>
  <tabstop>ptEditGroupBox</tabstop>
  <tabstop>ptNumSpinBox</tabstop>
  <tabstop>ptTypeComboBox</tabstop>
  <tabstop>panEditGroupBox</tabstop>
  <tabstop>panNumSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
